.github/workflows/deploy-firebase-test-cypress.yml

```diff
@@ -33,18 +31,28 @@ jobs:
     outputs:
       deployUrl: ${{ fromJson(steps.firebase-deploy.outputs.urls)[0] }}
 
+  run-dev-server:
+    name: Run dev server
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 'lts/*'
+      - run: npm ci && npm run dev
+
   cypress-run:
     name: Run Cypress Tests
-    needs: [build-and-preview]
+    needs: [build-and-preview, run-dev-server]
     runs-on: ubuntu-latest
     timeout-minutes: 90
     strategy:
       fail-fast: false
       matrix:
         browser: [chromium, edge]
-        containers: [1, 2, 3]
+        containers: [1, 2, 3, 4]
     env:
-      CYPRESS_BASE_URL: ${{ needs.build-and-preview.outputs.deployUrl }}
+      #      CYPRESS_BASE_URL: ${{ needs.build-and-preview.outputs.deployUrl }}
```

Comment:
Is it because of commenting on this that it is not deploying?

Reply:
Yes I don't expect the tests to actually deploy and run yet; still working on the GitHub action to interact with the dev server.